content/application-faq/application.md

```diff
@@ -43,9 +43,13 @@ A perfect application takes 30 days, if your supporting documents are accepted t
  around. If you are asked for tweaks to your supporting documents, expect 50-60 days.
 
 ## Do I need to list an address in Taiwan on the application?
-It’s entirely normal and accepted to apply with no local address. Once you get a place to stay,
+Yes. It’s entirely normal and accepted to apply with no local address. Once you get a place to stay,
  you can (must) update the details on your card, free of charge.
 
+ ## Can I use a temporary address?
```

Comment:
The address on an ARC is pretty important, so it might be better not to list that address if it's truly temporary. Important documents could be sent to that address and you might miss them.

Reply:
@fifieldt  I think that's an interesting question for person that are here on a visa exempts or staying at hotels/airbnb. 

Would the answer be: 

Yes, but we wouldn't recommend it. Important documents could be sent to that address and you might miss them.

Reply:
That's perfect

Reply:
Though, does everyone have the same understanding of 'temporary'? Should we refer to hotel/airbnb?